treeherder/services/pulse/consumers.py

```diff
@@ -187,8 +188,7 @@ class JointConsumer(PulseConsumer):
     thread, so we use multiple threads, one per consumer.
     """
 
-    queue_suffix = env("PULSE_QUEUE_NAME", default="queue")
-
+    queue_suffix = env("PULSE_QUEUE_NAME", default="queue_{}".format(uuid.getnode()))
```

Comment:
Can you please use `socket.gethostname()` instead?

Reply:
Okay